src/xmipp/bindings/python/envs_DLTK/xmipp_deepEMhancer.yml

```diff
@@ -2,12 +2,13 @@
 
 name: xmipp_deepEMhancer
 channels:
-  - anaconda
-  - defaults
   - rsanchez1369
   - conda-forge
 dependencies:
-  - python=3.6
-  - deepemhancer=0.12
-  - numba=0.45
+  - python=3.10
+  - deepemhancer #CUDA > 10 required. Drivers nvidia >= 450.80.02
+  - numba
+  - h5py
+  - libstdcxx-ng=14.2.0 # To be compatible with scipion env
```

Comment:
No version

Reply:
in your case? what is the error message?